index.html

```diff
@@ -787,96 +787,100 @@ <h3 id="alignment-between-ruby-annotation-and-base-text">Alignment between ruby
 <section>
 <h4 id="alignment-between-ruby-annotation-and-base-text-mono-ruby">Mono-ruby</h4>
 <p>In <a title="mono-ruby">mono-ruby</a>, 
-following three methods are mainly used for the relative positions of the ruby annotation and its base text.
-In this document, option 1 is picked. Characters of the ruby annotation are set solid.
+the following three methods are mainly used for the relative positions of the ruby annotation and its base text.
+This document uses the option 1. Characters of the ruby annotation are set solid.
 </p>
 <ol>
-  <li id="l20211116001">The center of the ruby annotation and of its base text are aligned in the inline direction.</li>
-  <li id="l20211116002">The beginning of the ruby annotation and of its base text are aligned in the inline direction.</li>
-  <li id="l20211116003">In principle, the beginning of the ruby annotation and of its base text are aligned in the inline direction.
-    If the ruby annotation is wider than its base text, 
-    follow <a href="#l20211116032">option 2</a> in [[[#overhang-surrounding-characters]]] with following direction as prioritized.
-    The results may depend on the character class of characters preceding and following.</li>
+ <li id="l20211116001">The ruby annotation's centre and base text is aligned in the inline direction.</li>
```

Comment:
Most of this document seems to be in American English (search for "center", for example), so we should probably be consistent and change "centre" to "center".

Reply:
Ah, thank you for pointing. This is from grammarly.com configuration of @kidayasuo at 4cb547b...